src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int supplySum = 0;
+            int buySum = 0;
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operation = parts[0];
+                int amount = Integer.parseInt(parts[1].trim());
+                if (operation.equals("supply")) {
+                    supplySum += amount;
+                } else if (operation.equals("buy")) {
+                    buySum += amount;
+                }
+            }
+            int result = supplySum - buySum;
+
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+                writeToFile(writer, "supply", supplySum);
+                writeToFile(writer, "buy", buySum);
+                writeToFile(writer, "result", result);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read/write data...", e);
+        }
```

Comment:
Ми повинні вміти ділити задачі на підзадачі і реалізовувати їх окремо та незалежно.
```suggestion
        String fileContent = readFileContent(fromFileName);
        String report = createReport(fileContent);
        writeToFile(toFileName, report);
```